spec/request/letters_request_spec.rb

```diff
@@ -127,8 +127,8 @@
       end
 
       before do
-        user.va_profile.edipi = '1005079999'
-        user.va_profile.participant_id = '600039999'
+        user.mpi.profile.edipi = '1005079999'
```

Comment:
Should be able to prevent this `user.mpi` call similar to the recommendation above:
`let(:mvi_profile) { build(:mvi_profile, edipi: 'blahblah', participant_id: 'blahblah') }`

`before { stub_mpi(mvi_profile) }`